search.html

```diff
@@ -286,6 +287,108 @@ <h2><a class="anchor" id="keyvalue"></a>Key-Value Pair Annotation Search</h2>
 
 <p>will return objects with a Key-Value pair annotation with the specific key: <span class="code-example">Temperature</span>.</p>
 
+<h2><a class="anchor" id="advanced"></a>Advanced Search</h2>
+
+<p>To understand basics of Search in OMERO, or to search for Key-Value pairs, please read also the <a href="http://help.openmicroscopy.org/search.html#tips"
```

Comment:
use ``https://help..`` instead of ``http://help``

Reply:
fixed by https://github.com/openmicroscopy/ome-help/pull/324/commits/0058e6687e7546397f993a11020b0f15953fdea5